tests/unit/utils/test_related_links_predictor.py

```diff
@@ -3,6 +3,28 @@
 from gensim.models import Word2Vec
 import pickle
 
+
+def rounded_related_links(links, digits):
+    """Takes a related links dictionary and return the same dictionary but with
+    the confidence score rounded off to the number of digits parameter.
+
+    This is used below because given the random process used to compute those
+    scores, they can slightly vary with no consequences on the final output. The
+    test below isn't meant to check that, so it uses this function to round off
+    the scores, so that it's the actual related links result that is compared.
+    """
```

Comment:
```suggestion
    """Takes a related links dictionary and return the same dictionary but with
    the confidence score rounded off to the number of digits parameter.

    This is used below because given the random process used to compute those
    scores, they can slightly vary with no consequences on the final output. The
    test below isn't meant to check that, so it uses this function to round off
    the scores, so that it's the actual related links result that is compared.
    
    The structure of `links` is as follows:
    
    ```
    links = {
        '03680a95-4cd4-46e6-b6d9-ec7aa5fb988e': [
            ['d9293a00-0e80-4039-b5cd-298b5153b2a3', 0.98765],
            ['eec5b7ac-2248-4ffc-a061-b95d9de988b3', 0.97654]
        ].
        '12344a95-4cd4-46e6-b6d9-ec7aa5dg234f': [
            ['d9293a00-0e80-4039-b5cd-298b5153b2a3', 0.98765],
            ['eec5b7ac-2248-4ffc-a061-b95d9de988b3', 0.97654]
        ]
    }
    ```
    """
```